server/methods/accounts/accounts.js

```diff
@@ -1057,16 +1057,27 @@ export function createFallbackLoginToken() {
  * @name accounts/setProfileCurrency
  * @memberof Methods/Accounts
  * @method
+ * @param {String} accountId - accountId of user to set currency of
+ * @param {String} currencyName - currency symbol to add to user profile
  * @summary Sets users profile currency
  */
-export function setProfileCurrency(currencyName) {
+export function setProfileCurrency(accountId, currencyName) {
```

Comment:
I'd rather see `accountId` added as second arg, which is then backwards compatible and you don't have to modify the places that call this.

This also needs to check to make sure the current user has permission to update accountId